### PR TITLE
more docs at submission

### LINKED ISSUE #261: Clarify the requirement for clear pkg documentation being available online  on submission
We have had a number of submissions recently which did not have enough documentation available online on submission  making it too difficult to evaluate package functionality. Some have more details in vignettes, but not made them available for viewing (eg through pkgdown docs) without first installing the package.

Proposal:

- Update the dev_guide instructions to authors:
  - Specify that, Ideally all pkgs should have pkgdown documentation. They should have a good reason not to that vice versa
  - In the case where it does not, enough detail should be in the README for editors to decide whether the package is in scope.
  - Make it very clear that this is their opportunity to make a case for their package. The more we understand about the package, the easier it will be to evaluate this as in scope or not.
- Add a check to the submission checklist about enough documentation being available on submission for an editor to determine functionality and scope and assess whether the package is in scope. 
- Add note in docs and blanket response for EiC to use if a submitted package does not meet the criteria

### PR DESCRIPTION
**Pull request by maelle**
Fix #261 

@annakrystalli 
- Do you remember in which submission you asked for more details? There was some good phrasing in it.
- Do you have good examples of READMEs?

### CODE REVIEW

**Comment by maelle** [comment]
@annakrystalli the preview now works. Would you have time to look at this PR this week? :pray: (no worries if not)

File: softwarereview_author.Rmd [hunk 1]
```diff
@@ -11,11 +11,12 @@ This concise guide presents the software peer review process for you as a packag
         [Journal of Open-Source Software](https://joss.theoj.org/) (JOSS), it should include a `paper.md` file describing the package. More detail on JOSS's requirements can be found [at their website](https://joss.theoj.org/about#author_guidelines).
     -   If you choose this option you should *not* submit your package to JOSS separately. It will be evaluated by JOSS based on the rOpenSci review.
 - Please consider the best time in your package's development to submit.  Your package should be sufficiently mature so that reviewers are able to review all essential aspects, but keep in mind that review may result in major changes.
-    - For any submission or pre-submission inquiry the README of your package should provide enough information about your package (goals, usage, similar packages) for the editors to assess its scope.
+    - For any submission or pre-submission inquiry the README of your package should provide enough information about your package (goals, usage, similar packages) for the editors to assess its scope. Even better, set up a pkgdown website for allowing assessment of functionality without installing the package.
```
Comment (annakrystalli) [inline softwarereview_author.Rmd:14]: I would slightly reword to emphasize the importance of assessing scope without installing:

> For any submission or pre-submission inquiry the README of your package should provide enough information about your package (goals, usage, similar packages) for the editors to assess its scope without having to install the package. Even better, set up a pkgdown website for allowing more detailed assessment of functionality online.

File: softwarereview_author.Rmd [hunk 2]
```diff
@@ -11,11 +11,12 @@ This concise guide presents the software peer review process for you as a packag
...
-    - At the submission stage, all major functions should be stable enough to be fully documented and tested.
+    - At the submission stage, all major functions should be stable enough to be fully documented and tested; the README should make a strong case for the package (the editors will read it to e.g. evaluate this as in scope or not.).
     - We strongly suggest submitting your package for review _before_ publishing on CRAN or submitting a software paper describing the package to a journal. Review feedback may result in major improvements and updates to your package, including renaming and breaking changes to functions.
     - Your package will continue to evolve after review, this book [provides guidance about the topic](#evolution).
+- In your initial (pre-)submission and README, strive to explain your package's functionality and aim assuming little to none domain knowledge. E.g. if you wrote something to interact with knitr engines, what's a knitr engine?
```
Comment (annakrystalli) [inline softwarereview_author.Rmd:19]: Minor style tweaks:

> In your initial (pre-)submission and README, strive to explain your package's functionality and aim assuming little to no domain knowledge from the reader.

File: softwarereview_editor.Rmd [hunk 3]
```diff
@@ -150,6 +150,27 @@ In details, the EiC plays the following roles
...
+In that case please ask for more details: even if the package is deemed out-of-scope, the package docs will have gotten better so we are fine asking for these efforts.
+
+Example text
+
+```markdown
+Hello <username> and many thanks for your submission.
+
+We are discussing whether the package is in scope and need a bit more information.
+
+Would you mind add more details and context to the README?
```
Comment (annakrystalli) [inline softwarereview_editor.Rmd:165]: > Would you mind adding more details and context to the README?

**Review by annakrystalli — COMMENTED** [review]
Done. All looks great, just made a few very minor/non-critical suggestions

**Comment by maelle** [comment]
Thank you!!

**Pull request merged**